### ISSUE TITLE
prepare_qat_fx  can't control enable\disable_fake_quant

### ISSUE THREAD

**Issue by hcqylymzc** [pytorch/pytorch#120983]
### 🐛 Describe the bug

I am using fx graph for QAT, I want to know the accuracy difference between FP and INT after QAT, refer to (https://discuss.pytorch.org/t/convert-back-to-unquantized-model/101622/9), I tried to turn on disable_fake_quant, but found that the accuracy did not change before and after the switch. At the same time, I found that the model after prepare_qat_fx(After using prepare_qat_fx, the observer exists in the model.) is not a subclass of FakeQuantizeBase, and the model does not have a disable/enable_fake_quant method and not have disable/enable_obersver method, and the accuracy becomes worse after convert. So I wonder whether the fake quant node is used after prepare_fx_qat. If it is not used, is it not training the fp model? How do I switch fake_quant_node and observer?

### Versions

torch 2.0

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel
